### ISSUE TITLE
[Idea] Make Autofish start/stoppable using command

### ISSUE THREAD

**Issue by Art0sh** [MCCTeam/Minecraft-Console-Client#1893]
### Prerequisites

- [X] I have read and understood the [user manual](https://github.com/MCCTeam/Minecraft-Console-Client/tree/master/MinecraftClient/config)
- [X] I made sure I am running the latest [development build](https://ci.appveyor.com/project/ORelio/Minecraft-Console-Client/build/artifacts)
- [X] I tried to [look for similar feature requests](https://github.com/MCCTeam/Minecraft-Console-Client/issues?q=is%3Aissue) before opening a new one

### Console Client Version

Latest

### Describe your problem

So when enabling AutoFish in the ini, I saw the bot "crawling" its inventory when first joining a server. 
While that probably is needed, it kind of messes with bungee servers, where you connect to a hub and then connect to your desired server, eg City or Skyblock. Now, when AutoFish crawls the bots inventory, it clicks the "server list" item which one can use to open a gui and click the server to connect to instead of using commands. That means, enabling AutoFish that early, it connects the bot to a random server and will probably not find the fishing rod it is probably searching for.

### Suggest a possible solution

Add command like autodrop on/off for AutoFish so one can turn the feature on when ready.

### Attach screenshot here (If applicable)

![image](https://user-images.githubusercontent.com/35751343/148316054-f45fd92c-cf7b-4006-aed6-d62411716d31.png)

**Comment by ReinforceZwei** [comment]
I don't understand your question. AutoFishing bot will not start itself until manual activated by holding fishing rod and sending `/useitem` command.

**Comment by Art0sh** [comment]
I think I have to apologize. Sorry. I almost went crazy bc of this, but it looks like I missed something. I almost was 100% sure I only enabled AutoFish before "it" started to happen. Join a server and every single item in inventory got clicked. But all this time, I forgot I had enabled AutoDrop too, although I thought it wasn't yet active since it didn't drop anything when it connected to the "final" server. Long story short; I missed AutoDrop, thought it was AutoFish, embarrassed myself bc of it.
